cylc/flow/scripts/cat_log.py

```diff
@@ -439,6 +439,7 @@ def main(parser, options, *args, color=False):
                 cmd.append('--remote-arg=%s' % quote(batchview_cmd))
             cmd.append(workflow_name)
             is_edit_mode = (mode == 'edit')
+            # TODO: Add Intelligent Host selection to this
```

Comment:
Happy to try and implement now, or declare it follow up. What do you think?

Reply:
Follow-up is fine, since this is a peripheral command.

Reply:
Can open a some-day issue with a permalink to this TODO or remove the TODO.